US_post_office/README.md

```diff
@@ -57,4 +57,4 @@ Sample input:
 }
 ```
 
-Run this workflow multiple times inside a fork to obtain mailing rates for different services (e.g. compare ground vs. priority vs express by calling this workflow asynchronously 3 times).
\ No newline at end of file
+Run this workflow multiple times inside a fork to obtain mailing rates for different services (E.g. compare ground vs. priority vs. express by calling this workflow asynchronously 3 times).
```

Comment:
Why should we use uppercase for `example given`? [Reference](https://english.stackexchange.com/questions/28172/when-a-sentence-starts-with-e-g-should-the-e-be-capitalized)

Reply:
I basically applied the rule of capitalization. Beginning of a sentebce